app/boards/shields/romac_plus/Kconfig.defconfig

```
@@ -0,0 +1,9 @@
+# Copyright (c) 2020 Pete Johanson, Richard Jones
+# SPDX-License-Identifier: MIT
+
+if SHIELD_ROMAC_PLUS
+
+config ZMK_KEYBOARD_NAME
+	default "RoMac_plus-v4"
```

This can be a "user friendly name", so can have spaces, etc. Perhaps:

```suggestion
	default "RoMac+ v4"
```

---

Ah okay. I updated it here. I think I couldn't use + in bluemicro or something so I applied the same logic. Nice and clean now. See [
0ce3686](https://github.com/zmkfirmware/zmk/pull/198/commits/0ce36865b2026c8739c01cebd3031180aa0f3b1f)